composer.json

```diff
@@ -76,6 +76,9 @@
                 "OM-1081 - Not detecting HTTPS behind a proxy": "https://raw.githubusercontent.com/OpenMage/composer-patches/74862e7a9fa29dd62c5c02d585d931ce9e3c5815/patches/OM-1081.patch",
                 "OM-2047 - Pass delimiter char to preg_quote": "https://raw.githubusercontent.com/OpenMage/composer-patches/74862e7a9fa29dd62c5c02d585d931ce9e3c5815/patches/OM-2047.patch",
                 "OM-2050 - Prevent checking known date codes": "https://raw.githubusercontent.com/OpenMage/composer-patches/74862e7a9fa29dd62c5c02d585d931ce9e3c5815/patches/OM-2050.patch"
+            },
+            "colinmollenhour/magento-redis-session": {
+                "Cm_RedisSession-Revert-190": "https://raw.githubusercontent.com/justinbeaty/composer-patches/7c39eba67913d0e97c121618ee0ba9d5c1588b42/patches/Cm_RedisSession-Revert-190.patch"
```

Comment:
I think we should have a Maho repo for all the patches, included the ones for ZF, would you like the idea?

Reply:
Probably a good idea yes, just in case someone force pushed or deleted the OM repo.

Reply:
done in https://github.com/MahoCommerce/maho/commit/73014072b0b52d20d7d573d8ce3c9371ccb4fae2

I'll now are your patch too

Reply:
```suggestion
                "Cm_RedisSession-Revert-190": "https://raw.githubusercontent.com/MahoCommerce/maho-composer-patches/0424e4b13657d2745931b9063020b005428b9334/Cm_RedisSession-Revert-190.patch"
```